<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>鼠标点击位置：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_posx">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string>1347</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_posy">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string>739</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>进程名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ed_title">
         <property name="text">
          <string>阴阳师-网易游戏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ed_class">
         <property name="text">
          <string>Win32Window</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>时间间隔(秒):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_watu_second">
         <property name="text">
          <string>31</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>鼠标点击次数:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ed_times">
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="tedit_logger">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>411</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="bt_once">
       <property name="text">
        <string>单次点击</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_getoff">
       <property name="text">
        <string>下班</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_getpos">
       <property name="text">
        <string>获取位置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_mouse_kick">
       <property name="text">
        <string>点击次数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>打工</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>次数:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_watu_times">
       <property name="text">
        <string>31</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_watu_kick">
       <property name="text">
        <string>设置次数</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
